Dockerfile.migrations

```diff
@@ -14,7 +14,8 @@ COPY <<load.sh /
 envsubst < /db/migrations/20230606172700__add_root_user.sql.envsubst > /db/migrations/20230606172700__add_root_user.sql
 rm /db/migrations/20230606172700__add_root_user.sql.envsubst
 
-export CONNECTION_STRING_RW=$\{CONNECTION_STRING_RW\}?sslmode=disable
+[[ "$CONNECTION_STRING_RW" == *sslmode* ]] && export CONNECTION_STRING_RW=$\{CONNECTION_STRING_RW/no-verify/require\}
```

Comment:
Is the 'export' required? Could imagine it works just as well without.

In general, this is an ugly hack, and thinking about it, we needed it only for the external DB at DO because we didn't install their certs. In the end, the whole string replace thingy shouldn't be necessary anymore

Reply:
`export` is removed with d22ef135c39d64875b1b96cd6553efa6dab36e22

`no-verify` might come back, if we choose to use TLS on some development instances, where we would not want/need to wrangle with (CA) certificates.